shared/params/config.go

```diff
@@ -112,6 +113,7 @@ var defaultBeaconConfig = &BeaconChainConfig{
 	ValidatorPrivkeyFileName:     "/validatorprivatekey",
 	WithdrawalPrivkeyFileName:    "/shardwithdrawalkey",
 	BLSPubkeyLength:              96,
+	HashCacheSize:                100000,
```

Comment:
why 100000?

Reply:
thought it wont be too aggressive on the memory. What would you suggest?